docs/changelog/changelog.rst

```diff
@@ -32,6 +32,8 @@ New Features
   If the full 3D distance is desired, set the ``3dDistance`` flag in the 
   configuration block of the :ref:`distance-task`.
   (`#360 <https://github.com/BYU-PCCL/holodeck/issues/360>`_)
+- Environment weather/time can be optionally configured with :ref:`scenarios`
+  (`#263 <https://github.com/BYU-PCCL/holodeck/issues/263>`_)
```

Comment:
Add a link to the docs page here (you'll need to add a anchor before the heading of weather.rst (```.. _weather:```) and then you can reference it with (``` :ref:`uav-agent` ```). Isn't restructured text beautiful?